src/plugins/vis_type_vega/public/line_vega_spec_expression.ts

```diff
@@ -0,0 +1,246 @@
+/*
```

Comment:
To follow existing naming conventions, can you rename this to `line_vega_spec_fn` or `line_vega_spec_function`? Also, because there are now multiple expression functions stored in `vis_type_vega` plugin, it might be useful to create an `expressions` dir to contain them all, similar to what's done in `visualizations` plugin